bcda/client/bluebutton.go

```diff
@@ -185,7 +185,7 @@ func (bbc *BlueButtonClient) GetClaim(mbi, jobID, cmsID, since string, transacti
 	mbiHash := HashIdentifier(mbi)
 	params := GetDefaultParams()
 	params.Set("mbi", mbiHash)
-	params.Set("excludeSAMHSA", "true")
+	//params.Set("excludeSAMHSA", "true")
```

Comment:
Why leave this in commented out?  It would be cleaner to just remove it.  Ditto for the other cases.